native-script.rb

```diff
@@ -0,0 +1,55 @@
+# coding: utf-8
+
+# Only the user can manually download and install Xcode from AppStore 
+puts "Installing Xcode. Please click on 'Get Xcode' on the following dialog and install Xcode from AppStore."
```

Comment:
Installing Xcode... Please, click 'Get Xcode' in the dialog that will appear next and install Xcode from the App Store.